ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron/Ledger.hs

```diff
@@ -159,40 +156,18 @@ instance ProtocolLedgerView ByronBlock where
   -- i.e., that delegate is allowed to issue a block in that very same slot.
   anachronisticProtocolLedgerView cfg (ByronLedgerState ls ss) slot =
```

Comment:
Should the docstring be updated?

Reply:
No, I think it's still accurate. The same bounds still apply. It's just that we return something from within those windows.